src/macros.rs

```diff
@@ -27,10 +27,10 @@ macro_rules! create_id {
 
 macro_rules! bail {
     ($msg:literal $(,)?) => {
-        return Err($crate::api_error!($msg))
```

Comment:
For the Rust newbie: what does this accomplish? 😄

Reply:
just because we have `return` somehow in the past it was working though not sure why :/